spec/services/flow/steps/provider_income/regular_incomes_step_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe Flow::Steps::ProviderIncome::RegularIncomesStep, type: :request do
+  let(:legal_aid_application) { create(:legal_aid_application) }
+
+  describe "#path" do
+    subject { described_class.path.call(legal_aid_application) }
+
+    it { is_expected.to eql providers_legal_aid_application_means_regular_incomes_path(legal_aid_application) }
+  end
+
+  describe "#forward" do
+    subject(:forward_step) { described_class.forward.call(legal_aid_application) }
+
+    context "when applicant has income types other than housing benefit" do
```

Comment:
Should housing benefit be in this description? It's not clear from the setup or the controller tests that housing benefit is involved here

Reply:
👍 was looking at income_types? on the legal_aid_application model which defines income_types? as transaction_types.credits.without_housing_benefits.any? but have removed.